doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -221,7 +221,7 @@ Keys samples
 Matter samples
 --------------
 
-|no_changes_yet_note|
+* Added support for nrf7001 DK to Matter Lock Sample.
```

Comment:
We don't really have a 7001 DK - better refer to it with its full name (emulation board)

Reply:
Ok, right. Is it better now?

Reply:
yes, to me at least